MyArrayList.java

```diff
@@ -1 +1,100 @@
+package com.company;
 
+public class MyArrayList<T> implements List<T> {
+
+    private static final int DEFAULT_CAPACITY = 10;
+    private int i = 0;
+    private Object[] array = new Object[DEFAULT_CAPACITY];
```

Comment:
do you think we could use type T instead of object here?

Reply:
Anyway when compiling generics turn into Objects, so we can make an array of Objects. Yes?